docs/api/modules/_operators_of_action_.md

```diff
@@ -209,6 +209,5 @@ This will ONLY grab actions that have just been successfully completed
 | `Rest` allowedTypes | `any`[] |
 
 **Returns:** `(Anonymous function)`
-
```

Comment:
Could you remove changes from this file? Just to skip it at all.

Reply:
done